android/app/src/main/java/app/organicmaps/widget/placepage/DirectionFragment.java

```diff
@@ -45,11 +46,15 @@ protected int getCustomTheme()
   public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState)
   {
     final View root = inflater.inflate(R.layout.fragment_direction, container, false);
+    final MwmActivity mwm_activity = (MwmActivity) requireActivity();
```

Comment:
That's not a python or rust )

```suggestion
    final MwmActivity mwmActivity = (MwmActivity) requireActivity();
```

Reply:
oops 😅 fixed